File: Algorithm_LevelOne/Reverse_String/Reverse_String.py
# Reverse String Python

class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        high = len(s) - 1
        low = 0
        while low < high:
            s[low], s[high] = s[high], s[low]
            low += 1
            high -= 1

File: Algorithm_LevelOne/Reverse_String/Reverse_String.cc
// Reverse String C++

#include <vector>

class Solution {
public:
    void reverseString(std::vector<char>& s) {
        int high = s.size() - 1;
        int low = 0;
        while (low < high) {
            char tmp = s[low];
            s[low++] = s[high];
            s[high--] = tmp;
        }
    }
};
